sklearn/utils/metadata_requests.py

```diff
@@ -10,8 +10,15 @@ class RequestType(Enum):
     UNREQUESTED = False
     REQUESTED = True
     ERROR_IF_PASSED = None
+    # this sentinel is used in `_metadata_request__*` attributes to indicate that a
+    # metadata is not present eventhough it may be present in the corresponding
+    # method's signature.
+    # TODO: I'm not happy with REMOVE, for now putting a Yoda style name here.
```

Comment:
Why not? Maybe UNUSED

Reply:
Cause I thought the semantics are not clear comparing "REMOVE" and "UNREQUESTED". I'm happy with "UNUSED"